### DISCUSSION TITLE
Selection of dimension for ATAC-seq

### DISCUSSION THREAD

**Discussion by pmcdil** [stuart-lab/signac#958] [Q&A] [answered]
Hi Signac team, first of all thanks for such a useful tool!

I am following the vignette https://satijalab.org/signac/articles/pbmc_multiomic.html for multiome (RNA+ATAC) dataset I have and I am not quite sure how do you select lsi components for ATAC part. 

`pbmc <- [FindMultiModalNeighbors](https://satijalab.org/seurat/reference/FindMultiModalNeighbors.html)(
  object = pbmc,
  reduction.[list](https://rdrr.io/r/base/list.html) = [list](https://rdrr.io/r/base/list.html)("pca", "lsi"), 
  dims.list = list(1:50, 2:40),
  modality.weight.name = "RNA.weight",
  verbose = TRUE
)` 

Here it's shown that for ATAC assay you use 40 components. I know selection of components is depending on your data but this selection is fairly easier when it comes to RNA assay since you can see gene sets in each PC and decide if these sets are getting noisier for higher PCs. However chromosomal locations are not like that and I am having difficulty to understand how to think to optimally select my components for ATAC assay. Could you explain the rationale behind this selection?

**Comment by timoast** [comment, marked as answer]
Please see https://satijalab.org/signac/articles/faq.html#how-should-i-decide-on-the-number-of-dimensions-to-use-
